compiler/rustc_target/src/target_features.rs

```diff
@@ -208,6 +213,11 @@ const X86_ALLOWED_FEATURES: &[(&str, Stability)] = &[
     ("avx512vnni", Unstable(sym::avx512_target_feature)),
     ("avx512vp2intersect", Unstable(sym::avx512_target_feature)),
     ("avx512vpopcntdq", Unstable(sym::avx512_target_feature)),
+    ("avxifma", Unstable(sym::avx512_target_feature)),
+    ("avxneconvert", Unstable(sym::avx512_target_feature)),
+    ("avxvnni", Unstable(sym::avx512_target_feature)),
+    ("avxvnniint16", Unstable(sym::avx512_target_feature)),
+    ("avxvnniint8", Unstable(sym::avx512_target_feature)),
```

Comment:
I don't see why the amx features are `-` but these are not? very odd choice on their part.

Reply:
not sure if these should be part of `avx512_target_feature` but don't know that they shouldn't be.

Reply:
They are just VEX coded versions of the AVX512 instructions, so I think they should be under the same feature flag, and we should stabilize them together.

Reply:
The peculiarity of their names stem from the naming conventions in both GCC and LLVM. Both name the amx features with `-`, and the AVX ones without that.

Reply:
yeah.
it sounds fine to me.